qsirecon/workflows/recon/build_workflow.py

```diff
@@ -198,6 +205,8 @@ def workflow_from_spec(available_anatomical_data, node_spec):
         "qsirecon_suffix": qsirecon_suffix,
         "params": parameters,
     }
+    if node_spec["action"] == "connectivity" and not config.execution.atlases:
```

Comment:
Great idea! Should we link to the docs where it says how to get the previous behavior with the `--atlases` flag?